decidim-core/app/models/decidim/user_moderation.rb

```diff
@@ -12,8 +12,6 @@ class UserModeration < ApplicationRecord
     scope :unblocked, -> { joins(:user).where(decidim_users: { blocked: false }) }
 
     delegate :organization, to: :user
-    scope :blocked, -> { joins(:user).where(decidim_users: { blocked: true }) }
```

Comment:
Could you please explain why you want to remove these two scopes from the `UserModeration` model?